docs/reference/msbuild-targets.md

```diff
@@ -169,6 +170,12 @@ You can also add the following metadata to your project reference:
 <PrivateAssets>
 ```
 
+### Deterministic
+
+When using `MSBuild -t:pack -p:Deterministic=true`, multiple invocations of the pack command will generate the exact same package.
+The output of pack is not affected by the ambient state of the machine. 
+Specifically, package internal psmdcp will be generated based on the package files, and the zip entries will be timestamped as 1/1/1980.
```

Comment:
```suggestion
Specifically, the package internal psmdcp will be generated based on the package files, and the zip entries will be timestamped as 1/1/1980.
```

Reply:
Removed the mentions of the psmdcp. Seems unnecessary in v1 :)